libp2p/protocols/secure/noise.nim

```diff
@@ -465,7 +465,7 @@ method handshake*(p: Noise, conn: Connection, initiator: bool): Future[SecureCon
         failedKey: PublicKey
       discard extractPublicKey(conn.peerInfo.peerId, failedKey)
       error "Noise handshake, peer infos don't match!", initiator, dealt_peer=conn.peerInfo.peerId, dealt_key=failedKey, received_peer=pid, received_key=remotePubKey
```

Comment:
error is for when the application cannot continue

Reply:
for that usually is "fatal", but I guess in chronicles it's error.. but true this should just be warn

Reply:
I might remove those when fixed anyway